mos_tests/murano/actions.py

```diff
@@ -106,19 +111,22 @@ def is_port_accesible():
         return wait(lambda: not negative == is_port_accesible(),
                     timeout_seconds=300, waiting_for='port access check')
 
-    def check_k8s_deployment(self, ip, port, timeout=3600, negative=False):
+    def check_k8s_deployment(self, ip, port, negative=False):
         def is_link_accesible():
-            tn = telnetlib.Telnet(ip, port)
-            tn.write('GET / HTTP/1.0\n\n')
-            buf = tn.read_all()
-            if len(buf) == 0:
-                return False
-            else:
-                tn.sock.sendall(telnetlib.IAC + telnetlib.NOP)
-                return True
+            try:
```

Comment:
I believe it's too complicated. is_link_accesible should return False if link is not accesible. You can compare with `not negative` in lambda within `wait`, like before